CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 ## 5.34.0 (Unreleased)
 
+NOTES:
+* compute: Updated field description of `connection_draining_timeout_sec`, `balancing_mode` and `outlier_detection` in `google_compute_region_backend_service` and `google_compute_backend_service`  to inform that default values will be changed in 6.0.0 ([#18399](https://github.com/hashicorp/terraform-provider-google/pull/18399))
+
+DEPRECATIONS:
+* vertexai: deprecated beta field `embedding_management` for `google_vertex_ai_feature_online_store` resource ([#18348](https://github.com/hashicorp/terraform-provider-google/pull/18348))
```

Comment:
This field is beta-only, so it might be worth restricting to just the beta changelog?